kobuki/e964bbb/e964bbb.bug

```diff
@@ -12,28 +12,48 @@ classification: "CWE-682: Incorrect Calculation #PHYSICAL"
 keywords: ['differential drive', 'velocity', 'driver', 'movement']
 system: kobuki
 severity: error
-links:
+links: []
+failure-codes:
+  - PROGRAMMING:CALCULATIONS
+fault-codes:
+  - SYSTEM:UNINTENDED-BEHAVIOUR
+  - SYSTEM:MOTION
 bug:
   phase: runtime
   specificity: robotics-specific
   architectural-location: application-specific code
   application: mobile robot
-  task: differential drive
+  task: locomotion
   subsystem: driver
-  package: yujinrobot/kobuki/kobuki_driver
-  languages: null
+  package:
+    - yujinrobot/kobuki/kobuki_driver
+  languages:
+    - C++
   detected-by: developer
   reported-by: contributor
   issue: https://github.com/yujinrobot/kobuki/issues/227
   time-reported: 2013-02-22T09:09:21Z
   reproducibility: always
-  trace:
+  trace: N/A
 fix:
   repo: https://github.com/yujinrobot/kobuki
   hash: e964bbb8700fb1a9b95c0cfe5a44d43321294d4f
-  pull-request: null
-  license: ['BSD']
-  fix-in: ['kobuki_driver/src/driver/diff_drive.cpp']
+  pull-request: N/A
+  license:
+    - 'BSD'
+  fix-in:
+    - 'kobuki_driver/src/driver/diff_drive.cpp'
   languages:
     - C++
-  time: 2013-02-25 (09:31)
+  time: 2013-02-25T09:31:00Z
```

Comment:
@ipa-hsd: I would not have expected a change like this to be included.

is the PR up-to-date?

Reply:
This is related to https://github.com/robust-rosin/robust/issues/370. The timezones and other issues in the bug files will be addressed in the subsequent PRs. Maybe I can rename this to `dummy.bug`

Reply:
I'm OK with this file being updated, but we should be *very* careful about changing timestamps.

So if you want to do this, I'd recommend looking up the actual UTC timestamp for the issue and/or PR. The time-machine repo has a script for that.

Reply:
I'd rather rename the file, since I am creating new PRs for sanitized bugs.

Reply:
BTW I just ran the script:
```
./get_issue_creation_date.py https://github.com/yujinrobot/kobuki/issues/227
2013-02-22T09:09:21Z
```
And for the commit (PR is N/A):
```
{
  "sha": "e964bbb8700fb1a9b95c0cfe5a44d43321294d4f",
  "node_id": "MDY6Q29tbWl0MzMzMjI0MTplOTY0YmJiODcwMGZiMWE5Yjk1YzBjZmU1YTQ0ZDQzMzIxMjk0ZDRm",
  "url": "https://api.github.com/repos/yujinrobot/kobuki/git/commits/e964bbb8700fb1a9b95c0cfe5a44d43321294d4f",
  "html_url": "https://github.com/yujinrobot/kobuki/commit/e964bbb8700fb1a9b95c0cfe5a44d43321294d4f",
  "author": {
    "name": "Younghun Ju",
    "email": "yhju83@gmail.com",
    "date": "2013-02-25T09:31:01Z"
  },
  "committer": {
    "name": "Younghun Ju",
    "email": "yhju83@gmail.com",
    "date": "2013-02-25T09:31:01Z"
  },
  "tree": {
    "sha": "9f5a77c075a2a19f1a862092b44724e224673422",
    "url": "https://api.github.com/repos/yujinrobot/kobuki/git/trees/9f5a77c075a2a19f1a862092b44724e224673422"
  },
  "message": "kobuki_driver : Fixed bug #227 in diff_drive module.",
  "parents": [
    {
      "sha": "35682ec3c3e7177373c66c48228ad58b2eb34aae",
      "url": "https://api.github.com/repos/yujinrobot/kobuki/git/commits/35682ec3c3e7177373c66c48228ad58b2eb34aae",
      "html_url": "https://github.com/yujinrobot/kobuki/commit/35682ec3c3e7177373c66c48228ad58b2eb34aae"
    }
  ],
  "verification": {
    "verified": false,
    "reason": "unsigned",
    "signature": null,
    "payload": null
  }
}
```
So it should be fine for this specific bug.

Reply:
Actually after our yesterday's discussion, I realized the kobuki package wasn't released when the issue was created. So the timestamp given by the script is not valid.